### DISCUSSION TITLE
How to get the window size created by eframe?

### DISCUSSION THREAD

**Discussion by LaBatata101** [emilk/egui#1141] [Q&A] [answered]
Is possible to get this information inside the update function?

**Comment by emilk** [comment, marked as answer]
You can use `ctx.input().screen_rect.size()`

**Reply by peterroe** [reply]
How can I get screen size? I don't sure if egui privide this feature. :)

**Reply by ngortheone** [reply]
`egui` itself does not provide this feature. The integration (most likely `winit`) does this
